fable-test-app/src/data/holidayData.tsx

```diff
@@ -0,0 +1,72 @@
+export const holidayData = [
+  {
+    id: 'federal',
+    name: 'Federal',
+    endpoint: 'https://canada-holidays.ca/api/v1/holidays?federal=true',
```

Comment:
If we're able to have an API BASE URL for the first path, we could revisit having this full URL in this file